.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,74 @@
+name: gh-pages
+
+on:
+  workflow_dispatch:
+    inputs:
+      project:
+        description: 'Repository'
+        required: true
+      ref:
+        description: 'Ref'
+        required: true
+  push:
+    branches:
+      - gh-pages
+
+jobs:
+  update-gh-pages: 
+    name: Update index.yaml
+    if: ${{ startsWith( github.ref, 'refs/heads/gh-pages') }}
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Update repo index 
+        run: |
+          echo "${{ secrets.CLONE_KEY }}" > id_rsa && chmod 400 id_rsa
+          trap "rm -f id_rsa" EXIT
+
+          helm repo index .
+          git diff index.yaml
+          git add index.yaml
+          git -c user.name="Ljubo Nikolic" -c user.email="ljubonikolic@gmail.com" commit -qam "Updating index.yaml"
```

Comment:
This will need change as well I assume?

Reply:
my creds should be replaced by repo owners, so we will wait for them to review these changes and then action will be required by them to add official repo creds 🙏